package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "react-starter-kit",
+  "homepage": "https://evita-m.github.io/ukol-moda/",
```

Comment:
ty github pages jsou v tomhle případě takový divný na setup. Musela bys mít zbuilděný web (npm run build) nahradý do gitu a pak někde v administraci nastavit tu cestu do dist/ složky nebo tak něco. 

Možná radši zkusit to nahrávat na netlify? tam by to mělo být přímočařejší